tests/unit/rpc/handlers/TxTests.cpp

```diff
@@ -136,6 +136,7 @@ constexpr auto kDEFAULT_OUT2 = R"({
     "close_time_iso": "2000-01-01T00:00:00Z",
     "validated": true
 })";
+constexpr static auto kINDEX = "E6DBAFC99223B42257915A63DFC6B0C032D4070F9A574B255AD97466726FC322";
```

Comment:
`static` is redundant here. Also, for future: we prefer `static constexpr auto` instead of `constexpr static auto`. Unfortunately there is no check that i know of to enforce it via clang-format or clang-tidy.